apps/api/src/app/message-template/usecases/create-message-template/create-message-template.usecase.ts

```diff
@@ -18,6 +18,10 @@ export class CreateMessageTemplate {
   ) {}
 
   async execute(command: CreateMessageTemplateCommand): Promise<MessageTemplateEntity> {
+    if ((command?.cta?.action as IMessageAction | undefined | '') === '') {
+      throw new ApiException('Please provide a valid CTA action');
+    }
```

Comment:
I do not like this solution but this would be the best way we could validate and alert the API user with a proper message, and this is because we do not support nested command validation [REF](https://github.com/novuhq/novu/pull/4306),  I will raise the topic with the team.